python/shotgun_model/data_handler_nav.py

```diff
@@ -146,10 +152,40 @@ class and the data payload from Shotgun should be provided via the
         num_modifications = 0
 
         new_uids = set()
+
+        # a list of sg data dicts to display items for
+        child_data = []
+
+        if item_path == self.__root_path:
+            self._log_debug("This is the root of the tree.")
+            parent_uid = None
+
+            if self.__include_root:
```

Comment:
I think I'm going to have to disagree with you guys on this one. Remember, the special item isn't the entire project view of the hierarchy. It's a special item that is only showing target entities attached at that level (no subfolders). The deselected state is the full project context and I think that's right still. Maybe try it out on the loader branch? I think it might be clearer once you've interacted with it? I'm willing to concede if you try it out and still feel strongly that it should select the special item.